src/rules/at-rule-no-unknown/__tests__/index.js

```diff
@@ -0,0 +1,94 @@
+import { testRule } from "../../../testUtils"
```

Comment:
While your fixing that empty line do you mind structuring the imports as:

```
import {
  messages,
  ruleName,
} from ".."
import rules from "../../../rules"
import { testRule } from "../../../testUtils"
```

For compatibility with [the `v7` branch](https://github.com/stylelint/stylelint/pull/1549). It'll save me having to tweak it later.